README.md

```diff
@@ -45,9 +45,13 @@ from the broker. This processor must implement
 use Swarrot\Processor\ProcessorInterface;
 use Swarrot\Broker\Message;
 
-class Processor implements ProcessorInterface {
-    public function process(Message $message, array $options) {
+class Processor implements ProcessorInterface
+{
+    public function process(Message $message, array $options)
+    {
         echo sprintf("Consume message #%d\n", $message->getId());
+
+        return true; // Continue processing other messages
```

Comment:
On top of documenting the return value, this might be a bit more future proof, if one day this lib requires php7 and you decide to enforce boolean as type hint for the return value.